function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
```

Comment:
Let's make this function dryer. We could write a ternary statement instead of spreading the code out into multiple lines.